### PR TITLE
Be setup

### PR DESCRIPTION
**Pull request by konradjosefsinger**

### CODE REVIEW

File: src/model/characterModel.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    });
+  },
+
+  getCharacterById: async (_id: string): Promise<Character | null> => {
+    return prisma.character.findUnique({
+      where: { id: _id },
+    });
+  },
+
+  // createCharacter: async (character: createNewCharacter): Promise<Character> => {
```
Comment (ElenaSestraSound) [inline src/model/characterModel.ts]: No commented code, 💩

File: src/model/frameDataModel.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  //   return prisma.frameData.findMany();
+  // },
+
+  getFrameDataById: async (_id: string): Promise<FrameData | null> => {
+    return prisma.frameData.findUnique({
+      where: { id: _id },
+    });
+  },
+
+  // createFrameData: async (frameData: createNewFrameData): Promise<FrameData> => {
```
Comment (ElenaSestraSound) [inline src/model/frameDataModel.ts]: 💩 comments

File: src/model/characterModel.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import { prisma } from '../prisma/client';
+import { Character, Move, FrameData } from './type';
+
+const dbCharacter = {
+
+  getAllCharacters: async (): Promise<Character[]> => {
+    return prisma.character.findMany();
+  },
+
+  getCharacterByName: async (name: string): Promise<Character | null> => {
```
Comment (ElenaSestraSound) [inline src/model/characterModel.ts:15]: We need:
- Save all the collection of characters
- Update one character
- Get all characters
- Get one character by name

File: src/model/frameDataModel.ts [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import { prisma } from '../prisma/client';
+import { FrameData } from './type';
+
+const dbFrameData = {
```
Comment (ElenaSestraSound) [inline src/model/frameDataModel.ts]: Not needed

File: src/scraper/scrapeController.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import { Request, Response, NextFunction } from 'express';
+
+
+/* *** GET *** */
+/* execute the scraping manually and populate the database for the first time or empty the database and populate it again */
+
+const scrapeManually = async (req: Request, res: Response, next: NextFunction) => {
```
Comment (ElenaSestraSound) [inline src/scraper/scrapeController.ts]: just scrape

File: src/model/characterModel.ts (hunk 1 above)
Comment (konradjosefsinger) [reply src/model/characterModel.ts]: commented Code deleted

File: src/model/characterModel.ts (hunk 3 above)
Comment (konradjosefsinger) [reply src/model/characterModel.ts:15]: Methods:
- getAllCharacters
- createCharacterCollection
- getCharacterByName
- getCharcterById
- updateOneCharacter

File: src/model/frameDataModel.ts (hunk 4 above)
Comment (konradjosefsinger) [reply src/model/frameDataModel.ts]: deleted

File: src/scraper/scrapeController.ts (hunk 5 above)
Comment (konradjosefsinger) [reply src/scraper/scrapeController.ts]: changed

**Pull request closed without merging**